### ISSUE TITLE
Should we rename dtis_cnt_minus_one to dts_cnt_minus_one

### ISSUE THREAD

**Issue by murillo128** [AOMediaCodec/av1-rtp-spec#176]
The `dtis_cnt_minus_one ` is defined as
```
dtis_cnt_minus_one: dtis_cnt_minus_one + 1 indicates the number of Decode targets present in the coded video sequence.
```

As the count is counting Decode Targets, shoudn't it be named `dts_cnt_minus_one ` instead? Same with `DtisCnt` to `DtsCnt`

**Comment by DanilChapovalov** [comment]
Nice catch!, Yes, we should.

**Comment by StephenBotzko** [comment]
Addressed in PR #177
